src/tests/system/tests/test_sssctl.py

```diff
@@ -150,7 +88,8 @@ def test_sssctl__handle_implicit_domain(client: Client):
 
 @pytest.mark.ticket(bz=1902280)
 @pytest.mark.topology(KnownTopology.LDAP)
-def test_sssctl__reset_cached_timestamps_to_reflect_changes(client: Client, ldap: LDAP):
+@pytest.mark.parametrize('fake', [""])
```

Comment:
What is the purpose of this 'fake' parameter? It's not used in the test.

Reply:
Sorry, misprint.